esmvaltool/recipes/recipe_autoassess_landsurface_permafrost.yml

```diff
@@ -55,7 +52,6 @@ diagnostics:
         obs_models: []
         start: 1993/12/01
         end: 2002/12/01
-        climfiles_root: '/gws/nopw/j04/esmeval/autoassess_specific_files/files'  # on JASMIN
```

Comment:
I would not remove this fully, but rather, comment out and add an inline comment saying eg this is the part that adds clima files, but they are not yet publicly available so this is commented out

Reply:
But the clim files are not needed by the permafrost recipe (only the soilmoisture recipe), so removal of this line is correct. See https://github.com/ESMValGroup/ESMValTool/issues/3065#issuecomment-1466643724

Reply:
ow my apologies, I was a bit too eager with the bullying :laughing: Nevermind me